lib/MusicBrainz/Server/Data/Collection.pm

```diff
@@ -264,6 +268,23 @@ sub delete_editor {
     );
 }
 
+sub set_collaborators {
+    my ($self, $collection_id, $collaborators) = @_;
+
+    $self->sql->begin;
+
+    $self->sql->do('DELETE FROM editor_collection_collaborator WHERE collection = ?', $collection_id);
+    $self->sql->insert_many(
```

Comment:
[`insert_many`](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-04-26/lib/Sql.pm#L159-L183) returns immediately if no array is given, but it seems to execute useless statements if empty array is given, or am I missing something? Ideally `insert_many` should be updated to skip empty array too, but if it is intended its input be checked here before calling it.

Reply:
Did that (I think, anyway!)

Reply:
Sorry, ignore my above comment, I confused array with array ref: `insert_many` takes an array rather than an array ref - it is  given a dereferenced array ref here - thus it makes no difference.

Reply:
Should I drop 0fe4da2ad47154e893fcac1c70b0635c29e7b1eb then?

Reply:
Sure